pom.xml

```diff
@@ -365,6 +366,12 @@
                 <version>${jline.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>javax.validation</groupId>
+                <artifactId>validation-api</artifactId>
+                <version>${javax-validation.version}</version>
```

Comment:
the schema generator relies on a newer version of javax-validation than we pull in by default.